config/environments/development.rb

```diff
@@ -15,6 +15,37 @@
   # Log error messages when you accidentally call methods on nil.
   config.whiny_nils = true
 
+  config.log_level = :debug
+
+  # Basic log config, for calls to Rails.logger.<level> { <message> }
+  config.logger = ::Logger.new(STDOUT)
+  config.logger.formatter = ->(severity, datetime, progname, msg) { "#{severity[0]} #{msg}\n" }
+
+  # Lograge config, overrides default instrumentation for logging ActionController and ActionView logging
+  config.lograge.enabled = true
+  config.lograge.custom_options = lambda do |event|
```

Comment:
I'd prefer the new lambda syntax. You are already using the new syntax in line 22, so maybe this could be changed for the sake of consistency?